benchmarks/winogrande/winogrande.py

```diff
@@ -0,0 +1,135 @@
+"""WINOGRANDE: An Adversarial Winograd Schema Challenge at Scale
+
+Keisuke Sakaguchi, Ronan Le Bras, Chandra Bhagavatula, Yejin Choi
+https://arxiv.org/pdf/1907.10641
+
+The prompt template is based on the multiple choice template in openai simple evals:
+https://github.com/openai/simple-evals/blob/main/mmlu_eval.py
+
+# run with default fewshot samples (5)
+inspect eval winogrande/winogrande.py
+
+# run with more or no fewshot samples
+inspect eval winogrande/winogrande.py -T fewshot=5
+inspect eval winogrande/winogrande.py -T fewshot=false
+
+# run with different subset of the dataset
+inspect eval winogrande/winogrande.py -T dataset_name=winogrande_l
+"""
+
+from typing import Dict
+
+from inspect_ai import Task, task
+from inspect_ai.dataset import Sample, hf_dataset
+from inspect_ai.model import GenerateConfig
+from inspect_ai.scorer import choice
+from inspect_ai.solver import (
+    multiple_choice,
+    system_message,
+)
+
+# Assumption: "A" and "B" are the option letters.
+ANSWER_TO_LETTER = {"1": "A", "2": "B"}
+BLANK_TAG = "[BLANK]"
+
+list(ANSWER_TO_LETTER.values())[0]
```

Comment:
Is this line of code intended to be here?

Reply:
Nope, that's a bug, removed it. Thanks for pointing it out !